lib/iris/tests/unit/fileformats/nc_load_rules/helpers/test_build_oblique_mercator_coordinate_system.py

```diff
@@ -0,0 +1,176 @@
+
+    class ParamTuple(NamedTuple):
+        id: str
+        nc_attributes: dict
+        expected_class: Type[CoordSystem]
+        coord_system_kwargs: dict
+
+    param_list: List[ParamTuple] = [
+        ParamTuple(
+            "default",
+            dict(),
+            ObliqueMercator,
+            dict(),
+        ),
+        ParamTuple(
+            "azimuth",
+            dict(azimuth_of_central_line=90),
+            ObliqueMercator,
+            dict(azimuth_of_central_line=90),
+        ),
+        ParamTuple(
+            "central_longitude",
+            dict(longitude_of_projection_origin=90),
+            ObliqueMercator,
+            dict(longitude_of_projection_origin=90),
+        ),
+        ParamTuple(
+            "central_latitude",
+            dict(latitude_of_projection_origin=45),
+            ObliqueMercator,
+            dict(latitude_of_projection_origin=45),
+        ),
+        ParamTuple(
+            "false_easting_northing",
+            dict(false_easting=1000000, false_northing=-2000000),
+            ObliqueMercator,
+            dict(false_easting=1000000, false_northing=-2000000),
+        ),
+        ParamTuple(
+            "scale_factor",
+            # Number inherited from Cartopy's test_mercator.py .
+            dict(scale_factor_at_projection_origin=0.939692620786),
+            ObliqueMercator,
+            dict(scale_factor_at_projection_origin=0.939692620786),
+        ),
+        ParamTuple(
+            "globe",
+            dict(semi_major_axis=1),
+            ObliqueMercator,
+            dict(ellipsoid=GeogCS(semi_major_axis=1, semi_minor_axis=1)),
+        ),
+        ParamTuple(
+            "combo",
+            dict(
+                azimuth_of_central_line=90,
+                longitude_of_projection_origin=90,
+                latitude_of_projection_origin=45,
+                false_easting=1000000,
+                false_northing=-2000000,
+                scale_factor_at_projection_origin=0.939692620786,
+                semi_major_axis=1,
+            ),
+            ObliqueMercator,
+            dict(
+                azimuth_of_central_line=90.0,
+                longitude_of_projection_origin=90.0,
+                latitude_of_projection_origin=45.0,
+                false_easting=1000000,
+                false_northing=-2000000,
+                scale_factor_at_projection_origin=0.939692620786,
+                ellipsoid=GeogCS(semi_major_axis=1, semi_minor_axis=1),
+            ),
+        ),
+        ParamTuple(
+            "rotated",
+            dict(grid_mapping_name="rotated_mercator"),
+            RotatedMercator,
+            dict(),
+        ),
+        ParamTuple(
+            "rotated_azimuth_ignored",
+            dict(
+                grid_mapping_name="rotated_mercator",
+                azimuth_of_central_line=45,
+            ),
+            RotatedMercator,
+            dict(),
+        ),
+    ]
+    param_ids: List[str] = [p.id for p in param_list]
```

Comment:
@trexfeathers Again, these could be defined outside the scope of the class.

Reply:
2bcee96993cc6f0ed21e2499136b600330177c26